app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt

```diff
@@ -42,7 +42,7 @@ fun ConversationScreen(
         onSendAttachment = { attachmentBundle -> conversationViewModel.sendAttachmentMessage(attachmentBundle) },
         onBackButtonClick = { conversationViewModel.navigateBack() },
         onDeleteMessage = conversationViewModel::showDeleteMessageDialog,
-        onCallStart = conversationViewModel::navigateToInitiatingCallScreen
+        onCallStart = { conversationViewModel.conversationId?.let { conversationViewModel.navigateToInitiatingCallScreen(it) } }
```

Comment:
i think there is no need no need to pass conversationViewModel.conversationId as ConversationId lives inside the ConversationViewModel

Reply:
Good point, thanks